z3c/dependencychecker/report.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+
+class Report(object):
+
+    def __init__(self, package):
+        self._database = package.imports
+
+    def __call__(self):
```

Comment:
I haven't seen yet how you call this class, but a regular method name *might* be clearer.
Not sure yet, I'm just writing it down as I go through the file.

Reply:
This is how it is used: https://github.com/reinout/z3c.dependencychecker/blob/gforcada-rewrite/z3c/dependencychecker/main.py#L22-L23

I'm leaving it as is, but if you want, we can change it for a name, no problem.

Reply:
It is the only place in the code where `__call__()` is used in this way, so a name is clearer. `.print_report()` or `.write_output()` or something like it?

Reply:
Done! I went for ``.print_report()``